Modules/Loadable/Segmentations/EditorEffects/Python/SegmentEditorEffects/AbstractScriptedSegmentEditorAutoCompleteEffect.py

```diff
@@ -66,6 +66,9 @@ def __del__(self):
         self.delayedAutoUpdateTimer.stop()
         self.observeSegmentation(False)
 
+    def cleanup(self):
```

Comment:
add a comment that describes why this is needed (if there is no comment and no failing test then the next developer may remove this call when he does some refactoring)